_episodes/01-start-existing-repo.md

```diff
@@ -0,0 +1,64 @@
+---
+layout: episode
+title: In-browser session
+teaching: 25
+exercises: 0
+objectives:
+  - See an existing repository in action.
+  - Browse the history.
+  - See the big picture first before we dive into details.
+---
+
+# In-browser session
+
+- We will explore and visualize an **existing Git repository** on GitHub and make small changes to it.
+- The goal of this episode is not to teach GitHub.
+- We will start out with a jam session on the web (work on existing repository).
+- Then we will practice scales on our local instruments (practice Git on our laptops).
+- Then (tomorrow) we regroup again and play orchestra music together.
+
+---
+
+## Why
+
+- Often our first contact with Git is an existing repository.
+- The existing repository is often on GitHub (but this demonstration applies equally well to GitLab or Bitbucket).
+- See the big picture before we focus.
+- Git commands that we will learn in the following session will make more sense.
```

Comment:
Remove last two lines.  Instead: 
- Most of the git commands we are about to learn can be done in the browser, which is a more intuitive way to get started.
- It's good to see the social aspect to know what our end goal is.